src/services/auth/authService.ts

```
@@ -153,6 +156,12 @@ export async function signup(params: SignupParams): Promise<SignupResult> {
 export async function signin(params: SigninParams): Promise<SigninResult> {
   const { identifier, passcode } = params;
   const user = await resolveUserByIdentifier(identifier);
+
+  if (user?.lockoutUntil && user.lockoutUntil > new Date()) {
+    logger.warn("Signin: account locked", { userId: user.id });
+    throw new Error("Account locked due to too many failed attempts. Please try again later.");
```

ESLint

[error] Replace `"Account·locked·due·to·too·many·failed·attempts.·Please·try·again·later."` with `⏎······"Account·locked·due·to·too·many·failed·attempts.·Please·try·again·later.",⏎····`

(prettier/prettier)